core/frontend/src/common/gltf/GltfSchema.ts

```diff
@@ -618,7 +631,7 @@ export namespace GltfStructuralMetadata {
     name?: string;
     description?: string;
     version?: string;
-    classes?: Class[];
+    classes?: any;
```

Comment:
Why did you change this? It was intended to reflect the [spec](https://github.com/CesiumGS/glTF/blob/3d-tiles-next/extensions/2.0/Vendor/EXT_structural_metadata/schema/class.schema.json).

Reply:
Ah, `Schema.classes` is a dictionary not an array per [spec](https://github.com/CesiumGS/glTF/blob/00578e29c554b9081373d0a6f916b48c4a8ce700/extensions/2.0/Vendor/EXT_structural_metadata/schema/schema.schema.json#L35). But the dictionary values are defined as `Class`es.

Reply:
the structural metadata schema does not hold an array of classes:

![image](https://github.com/user-attachments/assets/5903a20d-d19a-4383-a588-46284e7e1512)

Reply:
[Example](https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_structural_metadata#class).

```

  "extensions": {
    "EXT_structural_metadata": {
      "schema": {
        "classes": {
          "tree": {
            "name": "Tree",
            "description": "Woody, perennial plant.",
            "properties": {
              "species": { ... },
              "age": { ... },
              "height": { ... },
              "diameter": { ... }
```

Reply:
Here is what I changed:

```export interface Schema extends GltfProperty {
    id: string;
    name?: string;
    description?: string;
    version?: string;
    classes?: {
      [classId: string]: Class | undefined;
    };
    enums?: Enum[];
  }```